README.md

```diff
@@ -50,30 +50,125 @@ Secondary features :
+Once generated, the tokens must be scoped to the appropriate privilege levels
+required for the bot to function. Finally, you must ensure that you have
+subscribed the bot users to the list of events it will need to receive in order
+to operate.
 
-After restarting the bot, the bot user should now show up in your slack workspace.
+1) First, navigate to `api.slack.com/apps` and create a new slack app. Give it
+a name and select the workspace that it will serve, and then click create.
+
+2) Enable socket mode by clicking on the menu of the same name and then
+clicking on the toggle labelled "Enable Socket Mode". You'll be prompted to
+create an App Level Token which you must do next.
+
+2) App Level Token
+
+This token is used by the bot to access the websocket event feed in version 3
+of the Slack API. The toggle you enabled in the previous step, "Enable Socket
+Mode", is what turns on the event feed. Assuming you've not already created an
+App Level Token you'll now be prompted to do so. Place it into the config file
+under the entry `app_key`.
+
+3) Bot User Token
+
+This token is used by the bot to interact with the workspace and users via
+Slack's conversations API. In order to create this token you must first
+navigate to the OAuth & Permissions section of the Slack API control panel.
+
+Once there, take note of the greyed out "Install to Workspace" button. You need
+to add scopes to the bot user's token before you can install the bot. Scroll
+down to the bottom of this page and in the section labelled "Bot Token Scopes"
+add the following scopes.
+
+```
+app_mentions:read
+channels:history
+channels:join
+channels:manage
+channels:read
+chat:write
+chat:write.customize
+chat:write.public
+dnd:read
+emoji:read
+groups:history
+groups:read
+groups:write
+im:history
+im:read
+im:write
+links:read
+links:write
+mpim:history
+mpim:read
+mpim:write
+pins:read
+pins:write
+reactions:read
+reactions:write
+reminders:list
+reminders:write
+team:read
+users:read
+users:write
+```
+
+Next, return to the top of the page and install the bot user using the button
+from before which should no longer be greyed out. Now copy the "Bot User OAuth
+Access Token" (begins with `xoxb`) into the config file under the entry
+`api_key`. The bot should show up inside the workspace now. You can change its
+display name using the menu on the left labelled "Basic Information", if you
+want.
+
+4) Event subscription
+
+NOTE: Make sure you toggle "Enable Socket" (step 2) before attempting this!
+
+Once the tokens have been configured you will need to navigate to the Event
+Subscriptions panel of the slack API control panel and register the bot to
+receive the events it needs to operate. 
+
+First toggle the switch labelled "Enable Events", then use the picker to select
+at least the events below. They are probably all that you will need, but it's
+fine to select all of the event subscriptions here just to be on the safe side. 
```

Comment:
let's not say 'safe' since it's explicitly less secure :p let's call it simple.

Reply:
fair point that